flycheck.el

```diff
@@ -1364,7 +1364,7 @@ a default on its own."
                                   (flycheck-defined-checkers property))))
          (default (and default (symbol-name default)))
          (input (flycheck-completing-read
-                 prompt candidates default
+                 prompt candidates (or default (car candidates))
```

Comment:
The code below explicitly deals with the case of no checker selected case, so do we really need this?